.eslintrc

```diff
@@ -28,6 +28,8 @@
     "project": "./tsconfig.json"
   },
   "rules": {
+    "react/jsx-uses-react": "off",
+    "react/react-in-jsx-scope": "off",
```

Comment:
The commit comment explains: "There is an error loop between eslint and tsconfig, where one wants us
to import react, and the other does not."

I wondered about why this might be and ended up on https://github.com/jsx-eslint/eslint-plugin-react/#configuration. They mention:

> If you are using the [new JSX transform from React 17](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html#removing-unused-react-imports), extend [react/jsx-runtime](https://github.com/jsx-eslint/eslint-plugin-react/blob/c8917b0885094b5e4cc2a6f613f7fb6f16fe932e/index.js#L163-L176) in your eslint config (add "plugin:react/jsx-runtime" to "extends") to disable the relevant rules.

`jsx-runtime` explicitly disables these rules. Consequently I suggest that we import `plugin:react/jsx-runtime` instead of adding our own rules. This way we show that we are going with the community solution rather than our own coding standards.

Reply:
Thanks @kasperg ! 
We now use ```plugin:react/jsx-runtime``` instead ;)